ui/admin/app/components/worker-filter-generator/index.hbs

```diff
@@ -0,0 +1,196 @@
+          <Hds::Form::TextInput::Field
+            @value={{this.key}}
+            name='tag_key'
+            @width='320px'
+            {{on 'input' (set-from-event this 'key')}}
+            as |F|
+          >
+            <F.Label>{{t 'form.key.label'}}</F.Label>
+          </Hds::Form::TextInput::Field>
+        {{else}}
+          <Hds::Form::Select::Field
+            name='name_operator'
+            @value={{this.operator}}
+            @width='320px'
+            {{on 'change' (set-from-event this 'operator')}}
+            as |F|
+          >
+            <F.Label>{{t 'form.operator.label'}}</F.Label>
+            <F.Options>
+              <option value=''>
+                {{t
+                  'worker-filter-generator.filter-generator.operator.default'
+                }}
+              </option>
+              {{#each this.operatorOptions as |operator|}}
+                <option
+                  value={{operator}}
+                  selected={{eq operator this.operator}}
+                >
+                  {{#if (eq operator '==')}}
+                    {{operator}}
+                  {{else}}
+                    {{t
+                      (concat
+                        'worker-filter-generator.filter-generator.operator.'
+                        operator
+                      )
+                    }}
+                  {{/if}}
+                </option>
+              {{/each}}
+            </F.Options>
+          </Hds::Form::Select::Field>
+        {{/if}}
+      </F.Control>
+      <F.Control>
+        <Hds::Form::TextInput::Field
+          @value={{this.value}}
+          name='tag_value'
+          @width='320px'
+          {{on 'input' (set-from-event this 'value')}}
+          as |F|
+        >
+          <F.Label>{{t 'form.value.label'}}</F.Label>
+        </Hds::Form::TextInput::Field>
+      </F.Control>
+    </Hds::Form::Fieldset>
+  {{/let}}
+
+  {{#let (unique-id) (unique-id) as |labelId helpId|}}
+    <Hds::Form::Fieldset
+      aria-labelledby={{labelId}}
+      aria-describedby={{helpId}}
+      class='formatted-results'
+      @layout='vertical'
+      as |F|
+    >
+      <F.Legend id={{labelId}}>{{t
+          'worker-filter-generator.filter-generator.formatted-result.title'
+        }}</F.Legend>
+      <F.HelperText id={{helpId}}>{{t
+          'worker-filter-generator.filter-generator.formatted-result.description'
+        }}</F.HelperText>
+      <F.Control>
+        <Hds::Text::Body @tag='p' @weight='semibold'>{{t
+            'worker-filter-generator.filter-generator.formatted-result.label'
+          }}</Hds::Text::Body>
+        <div class='generated-results-container'>
+          <Hds::Form::TextInput::Base
+            readonly
+            id='generated-value'
+            name='generated_value'
+            @value={{this.generatedResult}}
+            @width='320px'
+          />
+          <Hds::Copy::Button @text='Copy' @targetToCopy='#generated-value' />
```

Comment:
Cannot think of why we went that route but happy to change it. I believe the copy fails because the generated results has no value to copy. When I changed it over to `@textToCopy` it doesn't fail but it does throw an error in the console.
```
index.js:153 Uncaught (in promise) Error: Assertion Failed: `hds-clipboard` modifier - either a `text` or a `target` argument is required
```

Reply:
Should we disable the button if there's no text then?

Reply:
After some thought and reviewing [HDS patterns](https://helios.hashicorp.design/patterns/disabled-patterns), I have decided to hide the button if there is no result to copy. Still happy to hear thoughts and suggestions for this though.

https://github.com/user-attachments/assets/86a8e91f-77b0-423f-a383-15d01df5db57

Reply:
Looks great!